shared/java/javastyle.gradle

```diff
@@ -53,7 +53,7 @@ if (!project.hasProperty('skipJavaFormat')) {
         format 'xml', {
             target fileTree('.') {
                 include '**/*.xml'
-                exclude '**/build/**', '**/build-*/**', '**/bin/**'
+                exclude '**/build/**', '**/build-*/**', '**/bin/**', '**/.idea/**', '**/.run/**'
```

Comment:
What's `.run`? I haven't seen it before.

Reply:
@auscompgeek .run is a folder for run configurations in IntelliJ

Reply:
Huh, I've only ever seen `.idea/runConfigurations/` used for that.

Reply:
I believe there's an option to store the run configurations outside of the ide configuration (with the intention of checking them in to source control I guess?). It seems to always set that option by default for me.